RFR-App/RFR/Measurements/MeasurementsViewModel.swift

```diff
@@ -148,8 +148,8 @@ class MeasurementsViewModel: ObservableObject {
             latitude: northWestCorner.coordinate.latitude - latitudeDistance/2,
             longitude: northWestCorner.coordinate.longitude + longitudeDistance/2
         )
-        let northSouthReach = southWestCorner.distance(from: northWestCorner)
-        let eastWestReach = northWestCorner.distance(from: northEastCorner)
+        let northSouthReach = min(southWestCorner.distance(from: northWestCorner), 0.1)
+        let eastWestReach = min(northWestCorner.distance(from: northEastCorner), 0.1)
```

Comment:
I don't know the context of this, but are you sure you want to cap the distance at 0.1 (max 0.1) and don't want to ensure a minimum distance or 0.1, which would be `max(..., 0.1)`?

Reply:
So you can see how tired I am all the time.